src/pybind/mgr/dashboard/module.py

```diff
@@ -103,18 +105,18 @@ def _configure(self):
         """
         server_addr = self.get_localized_module_option(
             'server_addr', get_default_addr())
-        ssl = self.get_localized_module_option('ssl', True)
-        if not ssl:
-            server_port = self.get_localized_module_option('server_port', 8080)
+        use_ssl = self.get_localized_module_option('ssl', True)  # type: ignore
+        if not use_ssl:
+            server_port = self.get_localized_module_option('server_port', 8080)  # type: ignore
         else:
             server_port = self.get_localized_module_option('ssl_server_port', 8443)
```

Comment:
Do we need `# type: ignore` here as well?

```suggestion
            server_port = self.get_localized_module_option('ssl_server_port', 8443)  # type: ignore
```